lib/v2/screens/dashboard/interactor/viewmodels/transactions_state.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:equatable/equatable.dart';
+import 'package:seeds/v2/datasource/remote/model/transaction_model.dart';
+
+class TransactionsState extends Equatable {
+  final bool isLoading;
```

Comment:
Please look at `PageState` 
You can use it to represent loading, success, etc

You need it in this state

Reply:
Done